miasm2/core/utils.py

```diff
@@ -83,31 +85,46 @@ def __setitem__(self, asked_key, value):
 
             # Bound can only be reached on a new element
             if (self._size >= self._max_size):
-                most_commons = [key for key, _ in self._counter.most_common()]
+                most_common = sorted(self._counter.iteritems(),
+                                     key=itemgetter(1), reverse=True)
```

Comment:
Just being curious: what are the advantages of importing and using `itemgetter` over using a `lambda` expression: `key=lambda item: item[1]` or `lambda (_, count): count`? Thanks!